<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>833</width>
    <height>753</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTableWidget" name="previewTableWidget">
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Enabled</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Action</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>From</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>To</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Preview</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btn_Export">
       <property name="toolTip">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Export summary - this writes a text file showing the contents of the preview window.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="text">
        <string>Export summary</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_OK">
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_Cancel">
       <property name="maximumSize">
        <size>
         <width>120</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_OK</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>process_data()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>641</x>
     <y>721</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>785</x>
     <y>731</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Export</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>export_summary()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>617</x>
     <y>731</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>previewTableWidget</sender>
   <signal>itemClicked(QTableWidgetItem*)</signal>
   <receiver>Form</receiver>
   <slot>preview_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>416</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>416</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>process_data()</slot>
  <slot>export_summary()</slot>
  <slot>preview_file()</slot>
 </slots>
</ui>
